### DISCUSSION TITLE
cyberspace-place.test not working

### DISCUSSION THREAD

**Discussion by goodBranchUC** [statamic/cms#6322] [Q&A] [answered]
windows 10
laravel 9.x
can't find help on this topic in discussion board or google.
i have tried php artisan serve still nothing.
any thoughts?

**Comment by joshuablum** [comment, marked as answer]
Hey @goodBranchUC,

You need to have Laravel Valet set up locally for the `.test` domains to work. You can find more on that [here](https://laravel.com/docs/9.x/valet). Valet is only available on macOS though but there is also a port for Windows which you can find out more about [here](https://github.com/cretueusebiu/valet-windows).

It's perfectly fine to use `php artisan serve` for this as well. If you run the command you can access the site at `localhost:8000` by default. With the `--host` and `--port` options you can change the defaults too.

Hope that helps :)

**Reply by goodBranchUC** [reply]
Thank you.
